### PR TITLE
8327753: Convert javax/swing/JOptionPane/8024926/bug8024926.java applet to main

### PR DESCRIPTION
**Pull request by prsadhuk**
Conversion of manual applet test to main based using PassFailJFrame manual framework

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8327753](https://bugs.openjdk.org/browse/JDK-8327753): Convert javax/swing/JOptionPane/8024926/bug8024926.java applet to main (**Bug** - P4)

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/18232/head:pull/18232` \
`$ git checkout pull/18232`

Update a local copy of the PR: \
`$ git checkout pull/18232` \
`$ git pull https://git.openjdk.org/jdk.git pull/18232/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 18232`

View PR using the GUI difftool: \
`$ git pr show -t 18232`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/18232.diff">https://git.openjdk.org/jdk/pull/18232.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/18232#issuecomment-1991639884)

### CODE REVIEW

File: test/jdk/javax/swing/JOptionPane/bug8024926.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual bug8024926
+ */
+public class bug8024926 {
+
+    private static final String INSTRUCTIONS = """
+                Verify that high resolution system icons are used
+                 in JOptionPane on HiDPI displays.
+                1) Run the test on Retina display or enable the Quartz Debug
```
Comment (prrace) [inline test/jdk/javax/swing/JOptionPane/bug8024926.java:46]: Quartz debug ? How ? And we don't even use Quartz.
The pass criteria is completely subjective. 
So this test looks like it needs to be better.
If you can't figure out a better test for this in less than a day just delete this useless thing.

Comment (prsadhuk) [reply test/jdk/javax/swing/JOptionPane/bug8024926.java:46]: I can remove the "Quartz debug" portion if it is not applicable, but as you know, it's a straight lift from the existing instructions...The reviewer that time did not object to this phrase it seems (was it used that time and subsequently revoked later?)
https://hg.openjdk.org/jdk9/jdk9/jdk/rev/4991a758abef

Also, the product fix has another test `test/javax/swing/JMenuItem/8031573/bug8031573.java`	which also put the same pass criteria that icons should be smooth and our manual testing team has not made any objections to the instructions of both test it seems..

Comment (prrace) [reply test/jdk/javax/swing/JOptionPane/bug8024926.java:46]: It looks like Quartz is still used in some native image handling code.
OK. If the instruction is harmless let it stay, but I doubt anyone does that.

**Review by prrace — APPROVED** [review]

**Pull request closed without merging**